VstsTasks/docs/overview.md

```diff
@@ -1,74 +1,119 @@
+
+**Template Parameters** (optional) - ARM template parameters to use. You can use any variable such as `$(Build.BuildNumber)` or `$(Release.ReleaseName)` when in a build or a release, respectively, for the `newVMName`. Similarly, you can create variables such as `User.Name` and `User.Password`, where the latter is marked as a _secret_.
+
+### Advanced Options
+The following advanced options can be specified on the task to help control the behavior of deployment:
+
+**Fail on artifact error** (optional) - Fail the task if any artifact fails to apply successfully.
+
+**Retry the deployment following any failure** (optional) - Retry the deployment when failing to create the lab VM or if any artifact fails to apply successfully.
+
+**Number of times to retry the deployment** (optional) - Number of times to retry the deployment when an error occurs, either while creating the lab VM or if any artifact fails to apply successfully.
+
+### Output Variables
+The task can produce the following outputs into corresponding variables:
+
+**Lab VM ID** - Variable to capture the created lab VM ID. Default is `labVMId`. The variable can be referred as `$(labVMId)` in subsequent tasks. You can use a name other than the default.
+
+## Delete VM
+![Delete VM](screenshots/azure-dtl-deletevm.png)
+The task allows you to delete a lab VM.
+### Input Parameters
+The task requires the following inputs: 
+
+**Azure RM Subscription** - Azure Resource Manager subscription to configure before running. 
+
+**Lab VM ID** - Resource ID of the lab VM to delete. Default is `$(labVMId)`. You can use any variable such as `$(labVMId)`, which is the output of calling **Create Azure DevTest Labs VM**, that contains a value in the form `/subscriptions/{subId}/resourceGroups/{rgName}/providers/Microsoft.DevTestLab/labs/{labName}/virtualMachines/{vmName}`.
+
+## Create Environment
+![Create Environment](screenshots/azure-dtl-createenvironment.png)
+The task allows you to create an Environment.
+### Input Parameters
+The task requires the following inputs:
+
+**Azure RM Subscription** - Azure Resource Manager subscription to configure before running.
+
+**Lab Name** - Azure DevTest Lab name.
+
+**Repository Name** - Repository friendly name where the environment ARM templates are stored.
+
+**Template Name** - Name of the environment ARM template.
+
+**Environment Name** - Name of the deployed template.
+
+**Parameters File** - The input parameters file. Use either a parameters file or the parameters.
+
+**Parameters** - The individual parameters.
+
+**Create output variables based on the environment template output?** - Checkbox to create output variables for the environment template. 
+
+### Output Variables
+The task can product the following outputs into corresponding variables
+
+**EnvironmentResourceId** - Resource ID of the environment created.
+
+**EnvironmentResourceGroupId** - Resource ID of the resource group for the environment created.

```

Comment:
Add the following at the end of the definition
> Usage: `$(<refName>.environmentResourceGroupId)`

Reply:
Corrected.